src/components/organisms/AdminVenueView/components/VenueRoomsEditor/VenueRoomsEditor.tsx

```diff
@@ -283,14 +283,14 @@ export const VenueRoomsEditor: React.FC<VenueRoomsEditorProps> = ({
             onClick={() => !selectedRoom && setSelectedRoom(room)}
           >
             <img
-              className={classNames("Container__room-image", {
-                "Container__room-image--disabled": !room.isEnabled,
+              className={classNames("VenueRoomsEditor__room-image", {
+                "VenueRoomsEditor__room-image--disabled": !room.isEnabled,
```

Comment:
It's worth keeping className calculation in separate var

Reply:
fixed in 24f391d2ee6cf6ad1d3364c161e287325e75ab44